{"uid":"7df1cd13d6df1613","name":"Update list name","fullName":"Trello UI Lists: Update list name","historyId":"e49678b56ac8fc8a7f7d860600e8906f","time":{"start":1657430623572,"stop":1657430650210,"duration":26638},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status\nfrom unknown error: unexpected command response\n  (Session info: chrome=103.0.5060.114)\nStacktrace:\n0   chromedriver                        0x0000000108460079 chromedriver + 4444281\n1   chromedriver                        0x00000001083ec403 chromedriver + 3970051\n2   chromedriver                        0x0000000108087038 chromedriver + 409656\n3   chromedriver                        0x00000001080743c8 chromedriver + 332744\n4   chromedriver                        0x0000000108073ac7 chromedriver + 330439\n5   chromedriver                        0x0000000108073047 chromedriver + 327751\n6   chromedriver                        0x0000000108071f16 chromedriver + 323350\n7   chromedriver                        0x000000010807244c chromedriver + 324684\n8   chromedriver                        0x000000010807f67e chromedriver + 378494\n9   chromedriver                        0x000000010807ff22 chromedriver + 380706\n10  chromedriver                        0x000000010808e5b3 chromedriver + 439731\n11  chromedriver                        0x000000010809247a chromedriver + 455802\n12  chromedriver                        0x000000010807277e chromedriver + 325502\n13  chromedriver                        0x000000010808e1fa chromedriver + 438778\n14  chromedriver                        0x00000001080ed11f chromedriver + 827679\n15  chromedriver                        0x00000001080da683 chromedriver + 751235\n16  chromedriver                        0x00000001080b0a45 chromedriver + 580165\n17  chromedriver                        0x00000001080b1a95 chromedriver + 584341\n18  chromedriver                        0x000000010843155d chromedriver + 4253021\n19  chromedriver                        0x00000001084363a1 chromedriver + 4273057\n20  chromedriver                        0x000000010843b16f chromedriver + 4292975\n21  chromedriver                        0x0000000108436dea chromedriver + 4275690\n22  chromedriver                        0x000000010841054f chromedriver + 4117839\n23  chromedriver                        0x0000000108450ed8 chromedriver + 4382424\n24  chromedriver                        0x000000010845105f chromedriver + 4382815\n25  chromedriver                        0x00000001084678d5 chromedriver + 4475093\n26  libsystem_pthread.dylib             0x00007fff204168fc _pthread_start + 224\n27  libsystem_pthread.dylib             0x00007fff20412443 thread_start + 15\n\n","statusTrace":"  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/gui/steps/lists_steps.py\", line 17, in step_impl\n    assert context.cards_page.check_if_list_is_created(context.list_name), \"Could not create the list\"\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/trello/gui/page_objects/board_page.py\", line 109, in check_if_list_is_created\n    return self.driver.is_element_found(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 181, in is_element_found\n    self.wait_for_presence_of_element_located(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 105, in wait_for_presence_of_element_located\n    return WebDriverWait(self.driver, TIMEOUT).until(EC.presence_of_element_located(locator))\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 81, in until\n    value = method(self._driver)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py\", line 64, in _predicate\n    return driver.find_element(*locator)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 857, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status\nfrom unknown error: unexpected command response\n  (Session info: chrome=103.0.5060.114)\nStacktrace:\n0   chromedriver                        0x0000000108460079 chromedriver + 4444281\n1   chromedriver                        0x00000001083ec403 chromedriver + 3970051\n2   chromedriver                        0x0000000108087038 chromedriver + 409656\n3   chromedriver                        0x00000001080743c8 chromedriver + 332744\n4   chromedriver                        0x0000000108073ac7 chromedriver + 330439\n5   chromedriver                        0x0000000108073047 chromedriver + 327751\n6   chromedriver                        0x0000000108071f16 chromedriver + 323350\n7   chromedriver                        0x000000010807244c chromedriver + 324684\n8   chromedriver                        0x000000010807f67e chromedriver + 378494\n9   chromedriver                        0x000000010807ff22 chromedriver + 380706\n10  chromedriver                        0x000000010808e5b3 chromedriver + 439731\n11  chromedriver                        0x000000010809247a chromedriver + 455802\n12  chromedriver                        0x000000010807277e chromedriver + 325502\n13  chromedriver                        0x000000010808e1fa chromedriver + 438778\n14  chromedriver                        0x00000001080ed11f chromedriver + 827679\n15  chromedriver                        0x00000001080da683 chromedriver + 751235\n16  chromedriver                        0x00000001080b0a45 chromedriver + 580165\n17  chromedriver                        0x00000001080b1a95 chromedriver + 584341\n18  chromedriver                        0x000000010843155d chromedriver + 4253021\n19  chromedriver                        0x00000001084363a1 chromedriver + 4273057\n20  chromedriver                        0x000000010843b16f chromedriver + 4292975\n21  chromedriver                        0x0000000108436dea chromedriver + 4275690\n22  chromedriver                        0x000000010841054f chromedriver + 4117839\n23  chromedriver                        0x0000000108450ed8 chromedriver + 4382424\n24  chromedriver                        0x000000010845105f chromedriver + 4382815\n25  chromedriver                        0x00000001084678d5 chromedriver + 4475093\n26  libsystem_pthread.dylib             0x00007fff204168fc _pthread_start + 224\n27  libsystem_pthread.dylib             0x00007fff20412443 thread_start + 15\n\n","statusTrace":"  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/gui/steps/lists_steps.py\", line 17, in step_impl\n    assert context.cards_page.check_if_list_is_created(context.list_name), \"Could not create the list\"\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/trello/gui/page_objects/board_page.py\", line 109, in check_if_list_is_created\n    return self.driver.is_element_found(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 181, in is_element_found\n    self.wait_for_presence_of_element_located(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 105, in wait_for_presence_of_element_located\n    return WebDriverWait(self.driver, TIMEOUT).until(EC.presence_of_element_located(locator))\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 81, in until\n    value = method(self._driver)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py\", line 64, in _predicate\n    return driver.find_element(*locator)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 857, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[{"name":"Given the user is logged in","time":{"start":1657430627591,"stop":1657430632974,"duration":5383},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"When the user goes to board \"CreateBoardExample\"","time":{"start":1657430632974,"stop":1657430644025,"duration":11051},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"And the user updates the list name \"ListTest\" to \"ListUpdate\"","time":{"start":1657430644026,"stop":1657430645246,"duration":1220},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Then the list is displayed","time":{"start":1657430645247,"stop":1657430645526,"duration":279},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status\nfrom unknown error: unexpected command response\n  (Session info: chrome=103.0.5060.114)\nStacktrace:\n0   chromedriver                        0x0000000108460079 chromedriver + 4444281\n1   chromedriver                        0x00000001083ec403 chromedriver + 3970051\n2   chromedriver                        0x0000000108087038 chromedriver + 409656\n3   chromedriver                        0x00000001080743c8 chromedriver + 332744\n4   chromedriver                        0x0000000108073ac7 chromedriver + 330439\n5   chromedriver                        0x0000000108073047 chromedriver + 327751\n6   chromedriver                        0x0000000108071f16 chromedriver + 323350\n7   chromedriver                        0x000000010807244c chromedriver + 324684\n8   chromedriver                        0x000000010807f67e chromedriver + 378494\n9   chromedriver                        0x000000010807ff22 chromedriver + 380706\n10  chromedriver                        0x000000010808e5b3 chromedriver + 439731\n11  chromedriver                        0x000000010809247a chromedriver + 455802\n12  chromedriver                        0x000000010807277e chromedriver + 325502\n13  chromedriver                        0x000000010808e1fa chromedriver + 438778\n14  chromedriver                        0x00000001080ed11f chromedriver + 827679\n15  chromedriver                        0x00000001080da683 chromedriver + 751235\n16  chromedriver                        0x00000001080b0a45 chromedriver + 580165\n17  chromedriver                        0x00000001080b1a95 chromedriver + 584341\n18  chromedriver                        0x000000010843155d chromedriver + 4253021\n19  chromedriver                        0x00000001084363a1 chromedriver + 4273057\n20  chromedriver                        0x000000010843b16f chromedriver + 4292975\n21  chromedriver                        0x0000000108436dea chromedriver + 4275690\n22  chromedriver                        0x000000010841054f chromedriver + 4117839\n23  chromedriver                        0x0000000108450ed8 chromedriver + 4382424\n24  chromedriver                        0x000000010845105f chromedriver + 4382815\n25  chromedriver                        0x00000001084678d5 chromedriver + 4475093\n26  libsystem_pthread.dylib             0x00007fff204168fc _pthread_start + 224\n27  libsystem_pthread.dylib             0x00007fff20412443 thread_start + 15\n\n","statusTrace":"  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"tests/gui/steps/lists_steps.py\", line 17, in step_impl\n    assert context.cards_page.check_if_list_is_created(context.list_name), \"Could not create the list\"\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/trello/gui/page_objects/board_page.py\", line 109, in check_if_list_is_created\n    return self.driver.is_element_found(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 181, in is_element_found\n    self.wait_for_presence_of_element_located(locator)\n  File \"/Users/runner/work/bootcamp04-trello/bootcamp04-trello/main/core/selenium/WebDriverUtils.py\", line 105, in wait_for_presence_of_element_located\n    return WebDriverWait(self.driver, TIMEOUT).until(EC.presence_of_element_located(locator))\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 81, in until\n    value = method(self._driver)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py\", line 64, in _predicate\n    return driver.find_element(*locator)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 857, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 435, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/runner/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},{"name":"And the list appears in the API","time":{"start":1657430650209,"stop":1657430650210,"duration":1},"status":"skipped","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[{"uid":"ccd35c9be09e2020","name":"<Scenario \"Update list name\">2022-07-10 05:24:09.567461","source":"ccd35c9be09e2020.png","type":"image/png","size":42365}],"parameters":[],"stepsCount":5,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"fixture.after.delete.board"},{"name":"tag","value":"fixture.after.delete.list"},{"name":"tag","value":"fixture.before.create.board"},{"name":"tag","value":"fixture.before.create.list"},{"name":"tag","value":"lists"},{"name":"tag","value":"lists_gui"},{"name":"tag","value":"update_list"},{"name":"feature","value":"Trello UI Lists"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["fixture.after.delete.board","fixture.before.create.board","fixture.after.delete.list","lists","fixture.before.create.list","update_list","lists_gui"]},"source":"7df1cd13d6df1613.json","parameterValues":[]}